src/ansible-cmdb

```diff
@@ -47,6 +47,10 @@ class Ansible(object):
             fd.close()
             try:
                 x = json.loads(''.join(s))
+                # for compatibility with fact_caching=jsonfile
+                # which omits the "ansible_facts" parent key added by the setup module
+                if not x.has_key('ansible_facts'):
```

Comment:
This breaks Python 3 compatibility. ``has_key()`` has been removed in 3.x, see: https://docs.python.org/3.1/whatsnew/3.0.html#builtins
Replace with ``if not 'ansible_facts' in x:`` which works in 2.x and 3.x.